### PR TITLE
Adding quota tests

### PR DESCRIPTION
**Pull request by lchayoun**
WIP

### CODE REVIEW

**Comment by lchayoun** [comment]
@marcosbarbero WDYT? change the exclusion patterns of coverage, added missing tests #18

File: spring-cloud-zuul-ratelimit-core/src/test/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/repository/ConsulRateLimiterTest.java [hunk 1]
```diff
@@ -39,4 +50,23 @@ public void setUp() {
         ObjectMapper objectMapper = new ObjectMapper();
```
Comment (marcosbarbero) [inline spring-cloud-zuul-ratelimit-core/src/test/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/repository/ConsulRateLimiterTest.java]: maybe this `objectMapper` is no longer needed?

Comment (lchayoun) [reply spring-cloud-zuul-ratelimit-core/src/test/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/repository/ConsulRateLimiterTest.java]: It is needed for the rest of the tests (from the base class)

Comment (marcosbarbero) [reply spring-cloud-zuul-ratelimit-core/src/test/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/repository/ConsulRateLimiterTest.java]: Got it! I'll merge it then.

**Pull request merged**